Tests/Unit/RealMiniAppViewTests.swift

```diff
@@ -24,7 +24,7 @@ class RealMiniAppViewTests: QuickSpec {
             context("when host app info is specified in plist") {
                 it("will add custom string in User agent") {
                     let miniAppView = RealMiniAppView(miniAppId: "miniappid-testing", versionId: "version-id", miniAppTitle: "", hostAppMessageDelegate: mockMessageInterface)
-                    expect(miniAppView.webView.customUserAgent).toEventually(contain("HOSTAPPNAME_AND_VERSION"), timeout: 3)
+                    expect(miniAppView.webView.customUserAgent).toEventually(contain("HOSTAPPNAME_AND_VERSION"), timeout: 6)
```

Comment:
I hope we never reach 6 seconds to achieve this test 😅, it happened to you ?

Reply:
Added 6 secs as a safe timeout. It failed for me like once/twice when I executed them in Local, maybe it takes time to get the value from plist I think.

Reply:
ok I see, then it's necessary, I think it depends on machine load